src/Stratis.Bitcoin.Features.Consensus.Tests/TimeSyncBehaviorAdjustedTimeTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Net;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Utilities;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    /// <summary>
+    /// Tests of <see cref="TimeSyncBehavior"/> and <see cref="TimeSyncBehaviorState"/> classes.
+    /// </summary>
+    public partial class TimeSyncBehaviorAdjustedTimeTest
+    {
+        /// <summary>
+        /// Time data is added this state. 
+        /// </summary>
+        private TimeSyncBehaviorState timesyncBehaviourState;
+
+        /// <summary>
+        /// This provides the adjusted time ready for assertion. 
+        /// </summary>
+        private IDateTimeProvider dateTimeProvider;
+        
+        /// <summary>
+        /// This is the result of the operation under test. 
```

Comment:
I don't think that this is a very representative comment. 
In general try to avoid comments like "it is used here and there" and "it is a result of" unless you are putting them into the remarks section. 
Summary comment should describe what this variable means and represents. In this case it represents a time offset between current machine's time and our view of the time on the network.

Reply:
yep makes sense re the "it is used here and there"  - but I don't think saying "it represents a time offset between current machine's time and our view of the time on the network." is something worth saying here. I think it is much clearer saying that the field exists as it is the result of the operation and what will be tested. 

Perhaps  I can rename to  adjustedOffsetResult ?